opr/params.py

```diff
@@ -83,9 +83,52 @@
     "GAU": "D", "GAC": "D",
     "GAA": "E", "GAG": "E",
     "GGU": "G", "GGC": "G",
-    "GGA": "G", "GGG": "G"
+    "GGA": "G", "GGG": "G",
+
+    "X": "INVALID",
+}
+
+CODONS_TO_AMINO_ACIDS_LONG = {
+    "UUU": "Phe", "UUC": "Phe",
+    "UUA": "Leu", "UUG": "Leu",
+    "UCU": "Ser", "UCC": "Ser",
+    "UCA": "Ser", "UCG": "Ser",
+    "UAU": "Tyr", "UAC": "Tyr",
+    "UAA": "Stop","UAG": "Stop",
+    "UGU": "Cys", "UGC": "Cys",
+    "UGA": "Stop","UGG": "Trp",
+
+    "CUU": "Leu", "CUC": "Leu",
+    "CUA": "Leu", "CUG": "Leu",
+    "CCU": "Pro", "CCC": "Pro",
+    "CCA": "Pro", "CCG": "Pro",
+    "CAU": "His", "CAC": "His",
+    "CAA": "Gln", "CAG": "Gln",
+    "CGU": "Arg", "CGC": "Arg",
+    "CGA": "Arg", "CGG": "Arg",
+
+    "AUU": "lle", "AUC": "lle", 
+    "AUA": "lle", "AUG": "Met",
+    "ACU": "Thr", "ACC": "Thr",
+    "ACA": "Thr", "ACG": "Thr",
+    "AAU": "Asn", "AAC": "Asn",
+    "AAA": "Lys", "AAG": "Lys",
+    "AGU": "Ser", "AGC": "Ser",
+    "AGA": "Arg", "AGG": "Arg",
+
+    "GUU": "Val", "GUC": "Val",
+    "GUA": "Val", "GUG": "Val",
+    "GCU": "Ala", "GCC": "Ala",
+    "GCA": "Ala", "GCG": "Ala",
+    "GAU": "Asp", "GAC": "Asp",
+    "GAA": "Glu", "GAG": "Glu",
+    "GGU": "Gly", "GGC": "Gly",
+    "GGA": "Gly", "GGG": "Gly",
+
+    "X": "INVALID",
```

Comment:
These two `X`s are not needed, but that's ok for this scope and they can be removed in the later refactor.